x-pack/plugins/security_solution/server/lib/detection_engine/README.md

```diff
@@ -46,6 +46,8 @@ server log [22:05:22.277] [info][status][plugin:alerting@8.0.0] Status changed f
 server log [22:05:22.270] [info][status][plugin:actions@8.0.0] Status changed from uninitialized to green - Ready
 ```
 
+Go into Kibana and load the default detection rules or manually create the `.siem-signals-elastic-default` index.
```

Comment:
Is this necessary? The following step of running the `hard_reset.sh` script will manage creating the signals index. 

https://github.com/elastic/kibana/blob/4584a8b570402aa07832cf3e5b520e5d2cfa7166/x-pack/plugins/security_solution/server/lib/detection_engine/scripts/hard_reset.sh#L22-L24